### PR TITLE
Update Observability BBEs for SLP9

### PR DESCRIPTION
**Pull request by madushajg**

### CODE REVIEW

File: examples/counter-metrics/counter_metrics.description [hunk 1]
```diff
@@ -1,5 +1,6 @@
 // Ballerina supports Observability out of the box, and Metrics is one of the three important aspect of the
-// Observability. To observe Ballerina code, the `--b7a.observability.enabled=true` property should be given when starting the service.
+// Observability. To observe Ballerina code, the `--observability-included` option and `--b7a.observability.metrics.enabled=true`
+// property should be given when starting the service.
```
Comment (nadundesilva) [inline examples/counter-metrics/counter_metrics.description:3]: We need to specifically mention which ones is the build time flag and runtime flag

Comment (nadundesilva) [reply examples/counter-metrics/counter_metrics.description:3]: Please update in all BBEs

Comment (madushajg) [reply examples/counter-metrics/counter_metrics.description]: Fixed with 0e9309f1eaf084500cf4d36a75141e8b1f1a2ce9

File: examples/counter-metrics/counter_metrics.bal [hunk 2]
```diff
@@ -2,20 +2,15 @@ import ballerina/http;
 import ballerina/io;
 import ballerina/log;
 import ballerina/observe;
+import ballerinax/prometheus as _;
 
 //Create a counter as a global variable in the service with optional field description.
```
Comment (praneesha) [inline examples/counter-metrics/counter_metrics.bal:7]:
```suggestion
//Create a counter as a global variable in the service with the optional field description.
```

File: examples/counter-metrics/counter_metrics.description [hunk 3]
```diff
@@ -1,5 +1,6 @@
 // Ballerina supports Observability out of the box, and Metrics is one of the three important aspect of the
```
Comment (praneesha) [inline examples/counter-metrics/counter_metrics.description:1]:
```suggestion
// Ballerina supports Observability out of the box and Metrics is one of the three important aspects of 
```

File: examples/counter-metrics/counter_metrics.description [hunk 4]
```diff
@@ -1,5 +1,6 @@
...
-// Observability. To observe Ballerina code, the `--b7a.observability.enabled=true` property should be given when starting the service.
+// Observability. To observe Ballerina code, the build time flag `--observability-included` and the runtime flag
+// `--b7a.observability.metrics.enabled=true` should be given when starting the service.
 // The developers can define and use metrics to measure their own logic. A counter is one type of metric that is
```
Comment (praneesha) [inline examples/counter-metrics/counter_metrics.description:4]:
```suggestion
// You can define and use metrics to measure your own logic. A counter is one type of the metrics that is
```

File: examples/counter-metrics/counter_metrics.description [hunk 5]
```diff
@@ -1,5 +1,6 @@
...
 // supported by default in Ballerina, and it is a cumulative metric that represents a single monotonically increasing
```
Comment (praneesha) [inline examples/counter-metrics/counter_metrics.description:5]:
```suggestion
// supported by default in Ballerina, and it is a cumulative metric that represents a single monotonically-increasing
```

File: examples/counter-metrics/tests/counter_metrics_test.bal [hunk 6]
```diff
@@ -10,7 +10,7 @@ function testFunc() {
     string response1 = "Order Processed!";
 
     // Send a GET request to the specified endpoint
```
Comment (praneesha) [inline examples/counter-metrics/tests/counter_metrics_test.bal:12]:
```suggestion
    // Send a GET request to the specified endpoint.
```

File: examples/gauge-metrics/gauge_metrics.bal [hunk 7]
```diff
@@ -2,21 +2,16 @@ import ballerina/http;
 import ballerina/io;
 import ballerina/log;
 import ballerina/observe;
+import ballerinax/prometheus as _;
 
 //Create a gauge as a global variable in the service with optional field description,
```
Comment (praneesha) [inline examples/gauge-metrics/gauge_metrics.bal:7]:
```suggestion
//Create a gauge as a global variable in the service with the optional field description,
```

File: examples/gauge-metrics/gauge_metrics.bal [hunk 8]
```diff
@@ -2,21 +2,16 @@ import ballerina/http;
...
 //and default statistics configurations = { timeWindow: 600000, buckets: 5,
```
Comment (praneesha) [inline examples/gauge-metrics/gauge_metrics.bal:8]:
```suggestion
//default statistics configurations = { timeWindow: 600000, buckets: 5,
```

File: examples/gauge-metrics/gauge_metrics.bal [hunk 9]
```diff
@@ -2,21 +2,16 @@ import ballerina/http;
...
 // percentiles: [0.33, 0.5, 0.66, 0.99] }.
```
Comment (praneesha) [inline examples/gauge-metrics/gauge_metrics.bal:9]:
```suggestion
// and percentiles: [0.33, 0.5, 0.66, 0.99] }.
```

File: examples/gauge-metrics/gauge_metrics.bal [hunk 10]
```diff
@@ -51,7 +46,7 @@ service onlineStoreService on new http:Listener(9090) {
         //counter instance.
```
Comment (praneesha) [inline examples/gauge-metrics/gauge_metrics.bal:46]:
```suggestion
        //Counter instance.
```

File: examples/gauge-metrics/gauge_metrics.description [hunk 11]
```diff
@@ -1,5 +1,6 @@
 // Ballerina supports Observability out of the box, and Metrics is one of the three important aspect of the
```
Comment (praneesha) [inline examples/gauge-metrics/gauge_metrics.description:1]:
```suggestion
// Ballerina supports Observability out of the box and Metrics is one of the three important aspects of the
```

File: examples/gauge-metrics/gauge_metrics.description [hunk 12]
```diff
@@ -1,5 +1,6 @@
...
-// Observability. To observe Ballerina code, the `--b7a.observability.enabled=true` property should be given when starting the service.
+// Observability. To observe Ballerina code, the build time flag `--observability-included` and the runtime flag
+// `--b7a.observability.metrics.enabled=true` should be given when starting the service.
 // The developers can define and use metrics to measure their own logic. A gauge is one type of metric that is
```
Comment (praneesha) [inline examples/gauge-metrics/gauge_metrics.description:4]:
```suggestion
// The developers can define and use metrics to measure their own logic. A gauge is one type of the metrics that is
```

File: examples/gauge-metrics/tests/gauge_metrics_test.bal [hunk 13]
```diff
@@ -10,19 +10,19 @@ function testFunc() {
     string response1 = "Order Processed!";
 
     // Send a GET request to the specified endpoint
```
Comment (praneesha) [inline examples/gauge-metrics/tests/gauge_metrics_test.bal:12]:
```suggestion
    // Send a GET request to the specified endpoint.
```

File: examples/tracing/tracing.bal [hunk 14]
```diff
@@ -1,15 +1,16 @@
...
 
     // Resource functions are invoked with the HTTP caller and the
     // incoming request as arguments.
-    resource function sayHello(http:Caller caller, http:Request req)
-                                returns error? {
+    resource function get sayHello(http:Caller caller, http:Request req)
+            returns error? {
         http:Response res = new;
 
         //Start a child span attaching to the system span generated.
```
Comment (praneesha) [inline examples/tracing/tracing.bal:16]:
```suggestion
        //Start a child span attaching to the generated system span.
```

File: examples/tracing/tracing.bal [hunk 15]
```diff
@@ -18,32 +19,32 @@ service hello on new http:Listener(9234) {
         //Start a new root span without attaching to the system span.
         int rootParentSpanId = observe:startRootSpan("MyRootParentSpan");
         // Some actual logic will go here, and for example we have introduced some delay with sleep.
```
Comment (praneesha) [inline examples/tracing/tracing.bal:21]:
```suggestion
        // Some actual logic will go here, and for example, we have introduced some delay with the sleep.
```

File: examples/tracing/tracing.bal [hunk 16]
```diff
@@ -18,32 +19,32 @@ service hello on new http:Listener(9234) {
...
-        runtime:sleep(1000);
+        runtime:sleep(1);
         //Start a new child span for the span `MyRootParentSpan`.
```
Comment (praneesha) [inline examples/tracing/tracing.bal:23]:
```suggestion
        //Start a new child span for the `MyRootParentSpan` span.
```

File: examples/tracing/tracing.bal [hunk 17]
```diff
@@ -18,32 +19,32 @@ service hello on new http:Listener(9234) {
...
         int childSpanId = check observe:startSpan("MyRootChildSpan", (),
                                                             rootParentSpanId);
         // Some actual logic will go here, and for example we have introduced some delay with sleep.
```
Comment (praneesha) [inline examples/tracing/tracing.bal:26]:
```suggestion
        // Some actual logic will go here, and for example, we have introduced some delay with the sleep.
```

File: examples/tracing/tracing.bal [hunk 18]
```diff
@@ -18,32 +19,32 @@ service hello on new http:Listener(9234) {
...
-        runtime:sleep(1000);
+        runtime:sleep(1);
         //Finish `MyRootChildSpan` span.
```
Comment (praneesha) [inline examples/tracing/tracing.bal:28]:
```suggestion
        //Finish the `MyRootChildSpan` span.
```

File: examples/tracing/tracing.bal [hunk 19]
```diff
@@ -18,32 +19,32 @@ service hello on new http:Listener(9234) {
...
         error? result = observe:finishSpan(childSpanId);
         if (result is error) {
-            log:printError("Error in finishing span", result);
+            log:printError("Error in finishing span", err = result);
         }
         // Some actual logic will go here, and for example we have introduced some delay with sleep.
```
Comment (praneesha) [inline examples/tracing/tracing.bal:33]:
```suggestion
        // Some actual logic will go here, and for example, we have introduced some delay with the sleep.
```

File: examples/tracing/tracing.bal [hunk 20]
```diff
@@ -18,32 +19,32 @@ service hello on new http:Listener(9234) {
...
-        runtime:sleep(1000);
+        runtime:sleep(1);
         //Finish `MyRootParentSpan` span.
```
Comment (praneesha) [inline examples/tracing/tracing.bal:35]:
```suggestion
        //Finish the `MyRootParentSpan` span.
```

File: examples/tracing/tracing.bal [hunk 21]
```diff
@@ -18,32 +19,32 @@ service hello on new http:Listener(9234) {
...
         result = observe:finishSpan(rootParentSpanId);
         if (result is error) {
-            log:printError("Error in finishing span", result);
+            log:printError("Error in finishing span", err = result);
         }
 
         //Some actual logic will go here, and for example we have introduced some delay with sleep.
```
Comment (praneesha) [inline examples/tracing/tracing.bal:41]:
```suggestion
        //Some actual logic will go here, and for example, we have introduced some delay with the sleep.
```

File: examples/tracing/tracing.bal [hunk 22]
```diff
@@ -18,32 +19,32 @@ service hello on new http:Listener(9234) {
...
-        runtime:sleep(1000);
+        runtime:sleep(1);
 
         //Finish the created child span `MyFirstLogicSpan`, which was attached to the system trace.
```
Comment (praneesha) [inline examples/tracing/tracing.bal:44]:
```suggestion
        //Finish the created child `MyFirstLogicSpan` span, which was attached to the system trace.
```

File: examples/tracing/tracing.description [hunk 23]
```diff
@@ -1,5 +1,6 @@
 // Ballerina supports Observability out of the box, and Tracing is one of the three important aspect of the
```
Comment (praneesha) [inline examples/tracing/tracing.description:1]:
```suggestion
// Ballerina supports Observability out of the box, and Tracing is one of the three important aspects of 
```

File: examples/tracing/tracing.description [hunk 24]
```diff
@@ -1,5 +1,6 @@
...
-// Observability. To observe Ballerina code, the `--b7a.observability.enabled=true` property should be given when starting the service.
+// Observability. To observe Ballerina code, the build time flag `--observability-included` and the runtime flag
+// `--b7a.observability.tracing.enabled=true` should be given when starting the service.
 // The developers can trace their code blocks and measure the time incurred during the actual runtime execution.
 // They can choose the hook their measurement with the default trace created or can create a completely new trace.<br/><br/>
```
Comment (praneesha) [inline examples/tracing/tracing.description:5]:
```suggestion
// They can choose to hook their measurement with the default trace created or can create a completely new trace.<br/><br/>
```

**Review by praneesha — APPROVED** [review]

**Pull request merged**